SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -566,6 +567,17 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment()
 		return;
 	}
 
+	if (bSchemaBuildError)
+	{
+		UE_LOG(LogSpatialGDKEditorToolbar, Warning, TEXT("Schema did not previously compile correctly, you may be running a stale build."));
+
+		EAppReturnType::Type Result = FMessageDialog::Open(EAppMsgType::YesNo, FText::FromString("Last schema generation failed or failed to run the schema descriptor. Schema will most likely be out of date, which may lead to undefined behavior. Are you sure you want to continue?"));
+		if (result == EAppReturnType::No)
```

Comment:
```suggestion
		if (Result == EAppReturnType::No)
```
How.... how is this compiling xD